contracts/pcv/balancer/BalancerLBPSwapper.sol

```diff
@@ -169,22 +171,14 @@ contract BalancerLBPSwapper is IPCVSwapper, OracleRef, Timed, WeightedBalancerPo
         5. Transfer remaining tokenReceived to tokenReceivingAddress
         @dev assumes tokenSpent balance of contract exceeds minTokenSpentBalance to kick off a new auction
     */
-    function swap() external override afterTime whenNotPaused {
-        (
-            uint256 spentReserves, 
-            uint256 receivedReserves, 
-            uint256 lastChangeBlock
-        ) = getReserves();
+    function swap() external override afterTime whenNotPaused onlyGovernorOrAdmin {
```

Comment:
@Joeysantoro Why did we move to permission this function? Was it due to MEV concerns?

Reply:
Reviewed report to refresh on this. All good from my side.